serverless/event_sources/knative-event-sources.adoc

```diff
@@ -7,22 +7,24 @@ toc::[]
 
 An _event source_ is an object that links an event producer with an event _sink_, or consumer. A sink can be a Knative service, channel, or broker that receives events from an event source.
 
-[id="knative-event-sources-creating"]
-== Creating event sources
-
 Currently, {ServerlessProductName} supports the following event source types:
 
 ApiServerSource:: Connects a sink to the Kubernetes API server.
 PingSource:: Periodically sends ping events with a constant payload. It can be used as a timer.
 SinkBinding:: Allows you to connect core Kubernetes resource objects such as a `Deployment`, `Job`, or `StatefulSet` with a sink.
+KafkaSource:: Connect a Kafka cluster to a sink as an event source.
+
+:FeatureName: Apache Kafka on {ServerlessProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
 
 You can create and manage Knative event sources using the **Developer** perspective in the {product-title} web console, the `kn` CLI, or by applying YAML files.
 
 * Create an xref:../../serverless/event_sources/serverless-apiserversource.adoc#serverless-apiserversource[ApiServerSource].
 * Create an xref:../../serverless/event_sources/serverless-pingsource.adoc#serverless-pingsource[PingSource].
-* Create a xref:../../serverless/event_workflows/serverless-sinkbinding.adoc#serverless-sinkbinding[SinkBinding].
-// Add Kafka once docs available
+* Create a xref:../../serverless/event_sources/serverless-sinkbinding.adoc#serverless-sinkbinding[SinkBinding].
+* Create a xref:../../serverless/event_sources/serverless-kafka-source.adoc#serverless-kafka-source[KafkaSource].
 
 [id="knative-event-sources-additional-resources"]
```

Comment:
Does this anchor need `_{context}` included, or is that not required for titles in assembly files? I thought I would just highlight this for your review in case it is a requirement.

Reply:
Just for modules afaik

Reply:
Ok thank you for clarifying that.